ui/summit-2024/src/pages/Leaderboard/Leaderboard.tsx

```diff
@@ -106,12 +113,32 @@ const Leaderboard: React.FC = () => {
               >
                 Leaderboard
               </Typography>
-              <AnimatedSwitch
-                defaultValue="Overall Votes"
-                optionA="Winners"
-                optionB="Overall Votes"
-                onClickOption={handleSwitch}
-              />
+              
+              {showRevealDate && ( // Shows once the voting has finished and before the reveal
```

Comment:
So both `showRevealDate` and `showWinners` are boolean values. When you use them with && in JSX when rendering, if they are false I think it will display a `0` randomly on the page. Safer (but not as nice) to use `showRevealDate ? (...) : undefined` (or null)

Reply:
`showWinners` part below unchanged

Reply:
My bad. Now it has been fixed